### PR TITLE
Refactor playingfield

### PR DESCRIPTION
**Pull request by LMiljak**
Many methods in playing field such as stopGameThread do something simple
like gameThread.stop(). All those methods that simply called something
in the gameThread/renderer are now gone. Classes that made use of these
methods now use getGameThread().someMethod() or getRenderer().

<organisation

### CODE REVIEW

**Comment by Diocruel** [comment]
Talked with Luka about it, should be fine. Code looks a lot cleaner now.

<layout, organisation

File: src/main/java/com/github/fishio/PlayingField.java [hunk 1]
```diff
@@ -574,4 +461,14 @@ public boolean isPlayerAlive() {
...
 		return false;
 	}
+	
+	/**
+	 * Sets the gameThread for this PlayingField.
+	 * 
+	 * @param gameThread
+	 * 		The game thread to set.
+	 */
+	public void setGameThread(GameThread gameThread) {
```
Comment (Taeir) [inline src/main/java/com/github/fishio/PlayingField.java]: The method setGameThread is never used and should be removed.

<organisation

Comment (LMiljak) [inline src/main/java/com/github/fishio/PlayingField.java]: Good point, I'm removing it now. I accidentally left that method there when trying to fix some powerup tests, but it turns out I didn't need it.

<organisation

**Comment by LMiljak** [comment]
Fixed it! I'm merging it.

<organisation

**Pull request merged**